flink-ml-lib/src/main/java/org/apache/flink/ml/classification/logisticregression/LogisticRegressionModel.java

```diff
@@ -150,21 +153,9 @@ public Row map(Row dataPoint) {
                 coefficient = modelData.coefficient;
             }
             DenseVector features = ((Vector) dataPoint.getField(featuresCol)).toDense();
-            Row predictionResult = predictOneDataPoint(features, coefficient);
-            return Row.join(dataPoint, predictionResult);
+            Tuple2<Double, DenseVector> predictionResult =
+                    LogisticRegressionModelServable.predictOneDataPoint(features, coefficient);
```

Comment:
Would it be more readable to instantiate an instance of `LogisticRegressionModelServable` and call `LogisticRegressionModelServable#transform(features)` here?